### PR TITLE
vocli documentation

### PR DESCRIPTION
**Pull request by randomshinichi**
@altergui @G10h4ck can you check if the README.md explains everything adequately? It's meant for people new to working with the vochain and its concepts, but not new to crypto.

### CODE REVIEW

File: cmd/vochaintest/vochaintest.go [hunk 1]
```diff
@@ -1739,7 +1739,7 @@ func testVocli(url, treasurerPrivKey string) {
 		log.Info("vocli txcost get NewProcess")
 		// get the initial txcosts. this is not used by the test, just for the
 		// human to read
-		if _, _, _, err := executeCommand(vocli.RootCmd, append([]string{"txcost", "get", aliceKeyPath}, stdArgs...), "", true); err != nil {
+		if _, _, _, err := executeCommand(vocli.RootCmd, append([]string{"txcost", "get"}, stdArgs...), "", true); err != nil {
 			log.Fatal(err)
 		}
 		_, _, _, err = executeCommand(vocli.RootCmd, append([]string{"txcost", "set", aliceKeyPath, "RegisterKey", "50"}, stdArgs...), "", true)
```
Comment (altergui) [inline cmd/vochaintest/vochaintest.go]: the changes to this file seem out of scope for "vocli documentation", split in a separate PR?

File: cmd/vocli/commands/helpers.go [hunk 2]
```diff
@@ -27,26 +27,8 @@ func writeKeyFile(filename string, k []byte) error {
...
-}
-
 func getKeysDir(root string) (string, error) {
-	dataDir, err := getDataDir(root)
-	if err != nil {
-		return "", err
-	}
-
-	return path.Join(dataDir, "keys"), nil
+	return path.Join(home, "keys"), nil
```
Comment (altergui) [inline cmd/vocli/commands/helpers.go]: same here, unrelated, split?

File: cmd/vocli/commands/txcost.go [hunk 3]
```diff
@@ -24,9 +24,9 @@ var txCostGetCmd = &cobra.Command{
 	NewProcess, RegisterKey, SendTokens, SetAccountInfo, SetProcessCensus, SetProcessQuestionIndex,
 	SetProcessResults, SetProcessStatus`,
 	Args: cobra.MatchAll(
-		cobra.MinimumNArgs(1),
+		cobra.MinimumNArgs(0),
 		func(cmd *cobra.Command, args []string) error {
-			for _, t := range args[1:] {
+			for _, t := range args {
```
Comment (altergui) [inline cmd/vocli/commands/txcost.go]: unrelated, split?

File: cmd/vocli/README.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+		Remember to run 'vocli account set <address>' afterwards to create an account for this key on the vochain.Your new key file will be locked with a password. Please give a password:
+
+Your new key was generated
+Public address of the key:   0x8bA764A28aff06335559B351027551DEf0C8EDF7
+Path of the secret key file: /home/shinichi/.dvote/keys/0x8bA764A28aff06335559B351027551DEf0C8EDF7-2022-6-21.vokey
+- As usual, please BACKUP your key file and REMEMBER your password!
+
+```
+
+One must additionally set an information string when setting an account. Normally an `ipfs://...` info-URI goes here, but this is unvalidated and can thus contain anything.
```
Comment (altergui) [inline cmd/vocli/README.md]: i'm not a native speaker but "unvalidated" sounds funny to me, why not say "this is currently never validated and can thus contain anything"

**Review by altergui — CHANGES_REQUESTED** [review]
disclaimer: i gave this a quick read, didn't actually test the commands described

File: cmd/vochaintest/vochaintest.go (hunk 1 above)
Comment (randomshinichi) [reply cmd/vochaintest/vochaintest.go]: I was hoping not to have two dependent PRs - oh well here's the other PR

**Review by p4u — APPROVED** [review]

**Review by altergui — APPROVED** [review]

**Comment by altergui** [comment]
(test was failing due to a race condition in vochaintest, totally unrelated to this PR. took 6 re-runs to make it pass :disappointed: i collected the evidence in https://github.com/vocdoni/vocdoni-node/issues/575 to investigate further)

**Comment by randomshinichi** [comment]
> (test was failing due to a race condition in vochaintest, totally unrelated to this PR. took 6 re-runs to make it pass disappointed i collected the evidence in #575 to investigate further)

thank you so much! It seems to only happen on the CI servers - on my machine, it seems to be more reliable. Maybe because in a virtualized environment, the processors are more busy. I remember the clocks in Windows Subsystem for Linux would always slowly get out of sync in a matter of days without NTP.

**Pull request merged**